grpc/wrappers.go

```diff
@@ -0,0 +1,51 @@
+package grpc
+
+import (
+	"errors"
+	"time"
+
+	"golang.org/x/net/context"
+
+	"github.com/letsencrypt/boulder/publisher"
+	pubPB "github.com/letsencrypt/boulder/publisher/proto"
+)
+
+// PublisherClientWrapper is a wrapper needed to satisfy the interfaces
+// in core/interfaces.go
+type PublisherClientWrapper struct {
+	inner   pubPB.PublisherClient
+	timeout time.Duration
```

Comment:
Ah, I was slightly confused by what the intention was here. Since we spawn a goroutine we don't really care if the parent method is cancelled, we still want to submit so having it work of a separate context makes sense.

For the comment on `ocsp-updater` I don't think spawning a new context makes sense though, the submission is a core part of that method, not something we split off, so it should be dependent on the parent context (which in this case is a `context.TODO` anyway which is exactly the same as `context.Background`).